README.md

```diff
@@ -105,6 +105,59 @@ steps:
         body: output
       })
 ```
+Specify terraform working directory and previous outputs can be used in subsequent steps for a concise comment on the pull request
```

Comment:
There should be an empty line before and after this line. I also think we should just replace the existing commenting example right above it instead of adding a new one. Commenting on pull requests isn't the focus of this GitHub action so I'd like to keep the commenting examples to a minimum in the README.

```suggestion

Outputs can be used in subsequent steps to comment on the pull request.

```

Reply:
Ok, reverted it back to your suggestion above

Reply:
Oh sorry! I meant that you can remove the existing example that's right above and replace it with your new example so that there's only one commenting example in the README.

Reply:
ok, I have replaced the existing example with my new example